src/components/Header/Header.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Header.css';
+
+class Header extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state={
+      users: [],
+      numOfUser: 0, 
+      currNum: 0
+    }
+  }
+
+  loadPeers() {
+    var togetherPeers = window.TogetherJS.require('peers');
+    var myName = togetherPeers.Self.name;
+    var myPeers = togetherPeers.getAllPeers();
+    var names = [];
+    names.push(myName);
+    myPeers.forEach(p => {
+      names.push(p.name);
+    });
+    this.setState({users: names});
+  }
+
+  endTurn() { 
+    this.setState({
+      currNum: (this.state.currNum+1)%3
```

Comment:
realized that I need to change 3

Reply:
Is this assuming there are three people on the page?

Reply:
yes, that's from developing, but I just updated that to a variable